adafruit_gps.py

```diff
@@ -291,10 +292,12 @@ def update(self) -> bool:
         # GP - GPS
         # GQ - QZSS
         # GN - GNSS / More than one of the above
+        # PG - Status of antenna extension
         if talker not in (b"GA", b"GB", b"GI", b"GL", b"GP", b"GQ", b"GN"):
             # It's not a known GNSS source of data
             # Assume it's a valid packet anyway
-            return True
+            # False: PGTOP. Status of antenna extension.
+            return True if data_type[:5] != b"PGTOP" else self._parse_pgtop(args)
```

Comment:
Suggestion for clarity:
```suggestion
            return True if data_type.endswith(b"PGTOP") else self._parse_pgtop(args)
```